### ISSUE TITLE
NewPipe crashes when changing resolution settings on WSA (Windows Subsystem for Android)

### ISSUE THREAD

**Issue by ckoever** [TeamNewPipe/NewPipe#7396]
Installed NewPipe on Windows Subsystem for Android via sideloaded F-Droid. 

Checklist
OK - I checked, but didn't find any duplicates (open OR closed) of this issue in the repo.
OK - I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
OK - This issue contains only one bug. I will open one issue for every bug report I want to file.

<details><summary><b>Steps to reproduce  </b></summary><p>

#### 1. Install Windows 11 build 22494.1000 (or above)
![image](https://user-images.githubusercontent.com/77546092/141376220-baf1be40-8223-4bba-a160-be648e97bebd.png)
*I own a "not supported" device/processor if it matters?*
![image](https://user-images.githubusercontent.com/77546092/141376297-e5ebb86b-b2f7-417b-8cad-a8862c483c34.png)

#### 2. Download WSA from [store.rg-adguard](https://store.rg-adguard.net)
ProductId *9p3395vx91nr*
Slow
Download *MicrosoftCorporationII.WindowsSubsystemForAndroid_1.8.32822.0_neutral_~_8wekyb3d8bbwe* ***.msixbundle*** (around 1.2GB)

#### 3. Install WSA via powershell 
```python
Add-AppPackage -path "...\MicrosoftCorporationII.WindowsSubsystemForAndroid_1.7.32822.0_neutral___8wekyb3d8bbwe.Msixbundle"
```

#### 4. Start Windows Subs..., activate developer mode and copy the IP
*You may open the "folders" to force start.*
![image](https://user-images.githubusercontent.com/77546092/141378407-cf05a015-8053-42bc-a399-85da5878df34.png)

![image](https://user-images.githubusercontent.com/77546092/141376897-075d226a-7ee5-43dd-afd9-5c20210d4eb2.png)

#### 6. Download [platform-tools](https://dl.google.com/android/repository/platform-tools-latest-windows.zip) and install F-Droid
Unzip, open cmd and run 
```python
...\adb.exe connect IPfromWSA
...\adb.exe install pathtoF-Droid.apk
```

#### 7. Install NewPipe via F-Droid and start it
![image](https://user-images.githubusercontent.com/77546092/141378131-402cc397-959c-4d5d-a9b7-def3dbe1428c.png)

#### 8. Watch any video and try to open the resolution settings
![image](https://user-images.githubusercontent.com/77546092/141374712-cc84a724-f153-4ca8-8b10-2e7fe80d4087.png)
</details>
<hr>

Actual behavior
When i try to open the resolution settings the UI breaks.

Exception
* __User Action:__ ui error
* __Request:__ ACRA report
* __Content Country:__ AT
* __Content Language:__ de-AT
* __App Language:__ de_AT
* __Service:__ none
* __Version:__ 0.21.13
* __OS:__ Linux Android 11 - 30

<details><summary><b>Crash log </b></summary><p>

```
android.view.WindowManager$BadTokenException: Unable to add window -- token android.view.ViewRootImpl$W@11424e0 is not valid; is your activity running?
	at android.view.ViewRootImpl.setView(ViewRootImpl.java:1071)
	at android.view.WindowManagerGlobal.addView(WindowManagerGlobal.java:409)
	at android.view.WindowManagerImpl.addView(WindowManagerImpl.java:109)
	at android.widget.PopupWindow.invokePopup(PopupWindow.java:1576)
	at android.widget.PopupWindow.showAsDropDown(PopupWindow.java:1423)
	at android.widget.ListPopupWindow.show(ListPopupWindow.java:722)
	at com.android.internal.view.menu.CascadingMenuPopup.showMenu(CascadingMenuPopup.java:432)
	at com.android.internal.view.menu.CascadingMenuPopup.show(CascadingMenuPopup.java:247)
	at com.android.internal.view.menu.MenuPopupHelper.showPopup(MenuPopupHelper.java:283)
	at com.android.internal.view.menu.MenuPopupHelper.tryShow(MenuPopupHelper.java:178)
	at com.android.internal.view.menu.MenuPopupHelper.show(MenuPopupHelper.java:142)
	at android.widget.PopupMenu.show(PopupMenu.java:230)
	at org.schabi.newpipe.player.Player.onQualitySelectorClicked(Player.java:3532)
	at org.schabi.newpipe.player.Player.onClick(Player.java:3664)
	at android.view.View.performClick(View.java:7448)
	at android.view.View.performClickInternal(View.java:7425)
	at android.view.View.access$3600(View.java:810)
	at android.view.View$PerformClick.run(View.java:28305)
	at android.os.Handler.handleCallback(Handler.java:938)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:223)
	at android.app.ActivityThread.main(ActivityThread.java:7664)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)

```
</details>
<hr>

Device info
WSA on
![image](https://user-images.githubusercontent.com/77546092/141376220-baf1be40-8223-4bba-a160-be648e97bebd.png)
![image](https://user-images.githubusercontent.com/77546092/141376297-e5ebb86b-b2f7-417b-8cad-a8862c483c34.png)

**Comment by mhmdanas** [comment]
Please don't remove the template when opening an issue.

**Comment by litetex** [comment]
Fun fact: This is the same error as on Samsung Dex (devices): https://github.com/TeamNewPipe/NewPipe/issues/7391

**Comment by litetex** [comment]
#6770 might be related

**Comment by yashpalgoyal1304** [comment]
waowww, didnt know this was technically called WSA.

pasting links for handy reference. will hide this comment myself.

* MS-docs: https://docs.microsoft.com/en-us/windows/android/wsa/
* github repo: https://github.com/WSA-Community
* WSA Tools (for sideloading): https://simonefranco.net/blog/2021/about-wsatools
* tangentially related: android on pinephones: https://liliputing.com/2021/11/glodroid-brings-android-12-to-the-pinephone-sort-of.html

**Comment by alex-sandro92** [comment]
I reproduced this bug and attached here the crash log, seems that there's a problem opening a popup window

[NewPipe_crash_log.txt](https://github.com/TeamNewPipe/NewPipe/files/7870228/NewPipe_crash_log.txt)

**Comment by litetex** [comment]
Issue was merged (with similar ones) into #7865.
Please go on with the conversation inside there.
